### ISSUE TITLE
Create a demo issue for Sunday meeting

### ISSUE THREAD

**Issue by Sparky-code** [hackforla/website#3164]
### Overview

Showcasing the templates for issue creation. Walk though to visualize the process.

### Action Items

- [ ] Doing the first goal that helps you understand the particulars of the isse
- [ ] do x see that y fails 
- [ ] do z see that a fails
- [ ] document how that failure occurs
- [ ] propose a sulution

### Resources/Instructions

good

### Is there a dependency?

Yes

### If Yes, please explain

also good
